models/keep3r_network/keep3r_network_token_price_legacy.sql

```diff
@@ -0,0 +1,11 @@
+{{ config( 
+  schema = 'keep3r_network',
+  alias = alias('token_price', legacy_model=True),
```

Comment:
minor: looks like dummy legacy file here is missing `s` at the end. dunesql version uses `token_prices`
this applies to both alias here and file name